eZ/Publish/SPI/Search/Document.php

```diff
@@ -31,6 +31,27 @@ class Document extends ValueObject
     public $languageCode;
 
     /**
+     * Denotes that document's translation is always available
```

Comment:
Updated in https://github.com/ezsystems/ezpublish-kernel/commit/145069503ecfdca2944e1bd53f3f077ca562c080